src/lib.rs

```diff
@@ -893,3 +898,13 @@ fn getenv_unwrap(v: &str) -> String {
 fn fail(s: &str) -> ! {
     panic!("\n{}\n\nbuild script failed, must exit now", s)
 }
+
+// Returns the major, minor and patch versions of cmake
+fn cmake_version() -> (u8, u8, u8) {
+    let cmd = Command::new("cmake").arg("--version").output().unwrap();
+    let version = String::from_utf8_lossy(&cmd.stdout);
+    let version: Vec<&str> = version.split_whitespace().collect();
+    let version: Vec<&str> = version[2].split(".").collect();
+    let version: Vec<u8> = version.iter().map(|c| c.parse().unwrap()).collect();
```

Comment:
I think these `.unwrap()` will want to go away and we'll want to return an `Option` from this function to avoid panicking if something goes wrong here.

Reply:
I think I can remove this if things work out with the MP flag.
Looking at the microsoft docs, the m flag doesn't seem to be listed:
https://docs.microsoft.com/en-us/cpp/build/reference/compiler-options-listed-alphabetically?view=msvc-160